src/Page/Story/Data.elm

```diff
@@ -13,9 +14,9 @@ type alias Story =
     , slug : String
     , maybeLocation : Maybe String
     , maybeGroupOrIndividual : Maybe String
-    , maybeImages : Maybe (List Page.Shared.View.Image)
+    , maybeImages : Maybe (List Image)
     , fullTextMarkdown : String
-    , relatedGuideList : List Page.Shared.View.GuideTeaser
+    , relatedGuideList : List GuideTeaser
```

Comment:
request change: show import location for internal code rather than destructuring